bpf/lib/common.h

```diff
@@ -597,7 +599,7 @@ enum {
 #define DROP_INVALID_EXTHDR	-156
 #define DROP_FRAG_NOSUPPORT	-157
 #define DROP_NO_SERVICE		-158
-#define DROP_UNUSED8		-159 /* unused */
```

Comment:
@dylandreimerink This drop reason wasn't added in `flow.proto` and `drop.go`.

Under normal circumstances, we shouldn't reuse any of these, since renaming a proto field/type causes a backwards-incompatible change. We're lucky that in this case, drop reason `159` is actually missing from the proto as well as from drop.go. :sweat_smile: 

```
    SERVICE_BACKEND_NOT_FOUND = 158;
    NO_TUNNEL_OR_ENCAPSULATION_ENDPOINT = 160;
```

In any case, I'm marking all unused ones as deprecated in https://github.com/cilium/cilium/pull/29482.

cc @rolinh